server/cli/cert.go

```diff
@@ -40,7 +40,7 @@ individual files in the directory specified by --certs (required).
 // runCreateCACert generates key pair and CA certificate and writes them
 // to their corresponding files.
 func runCreateCACert(cmd *cobra.Command, args []string) {
-	err := security.RunCreateCACert(Context.Certs)
+	err := security.RunCreateCACert(Context.Certs, Context.KeySize)
```

Comment:
Per my other comment, just define a `var keySize int` global in this file and hook it up to the command line flag parsing.